### PR TITLE
[sc-4963] command time took metric

### PR DESCRIPTION
**Pull request by steb-veksler**

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -168,9 +174,19 @@ func validateAuthentication(cmd *cobra.Command, args []string) error {
...
 
 func ExecuteContext(ctx context.Context) error {
+	start := time.Now()
 	err := RootCmd.ExecuteContext(ctx)
 
+	sentryCommandContext := sentry_utils.NewCommandContext(start)
+	sentryCommandContext.SetOnCurrentScope()
+
 	if err == nil {
+		sentry.CaptureMessage(fmt.Sprintf("%s executed successfully", sentryCommandContext.Name))
```
Comment (noamApps) [inline cmd/root.go:184]: very very nice

Comment (noamApps) [reply cmd/root.go:184]: where is the sentryCommandContext.Name defined ?

File: cmd/root.go [hunk 2]
```diff
@@ -168,9 +174,19 @@ func validateAuthentication(cmd *cobra.Command, args []string) error {
 }
 
 func ExecuteContext(ctx context.Context) error {
```
Comment (noamApps) [inline cmd/root.go:176]: is this called everytime at the end of a command if I understand correctly?

**Review by noamApps — COMMENTED** [review]
looks great, just some questions becuase it relies on sentry mechanisms Im not familiar with and looks like they are given in this pr

File: cmd/root.go (hunk 1 above)
Comment (steb-veksler) [reply cmd/root.go:184]: https://github.com/groundcover-com/cli/blob/ab5d3c73cfbb16a98e20dc83c461e5b1e080b732/cmd/root.go#L96
https://github.com/groundcover-com/cli/blob/ab5d3c73cfbb16a98e20dc83c461e5b1e080b732/pkg/sentry/context.go#L30

File: cmd/root.go (hunk 2 above)
Comment (steb-veksler) [reply cmd/root.go:176]: https://github.com/groundcover-com/cli/blob/ab5d3c73cfbb16a98e20dc83c461e5b1e080b732/main.go#L44

called as the main process

**Review by noamApps — APPROVED** [review]

**Pull request merged**